pkg/cluster/px/px.go

```diff
@@ -345,6 +346,27 @@ func (ops *pxClusterOps) runDockerPuller(imageToPull string) error {
 					Labels: labels,
 				},
 				Spec: corev1.PodSpec{
+					Affinity: &corev1.Affinity{
+						NodeAffinity: &corev1.NodeAffinity{
+							RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+								NodeSelectorTerms: []corev1.NodeSelectorTerm{
+									{
+										MatchExpressions: []corev1.NodeSelectorRequirement{
+											{
+												Key:      pxEnableLabelKey,
+												Operator: corev1.NodeSelectorOpNotIn,
```

Comment:
Dont we have other "non" true values like uninstall/remove?

Reply:
`px/enabled=remove` eventually uses `px/enabled=false` as a way to prevent px from running on the node.

`px/enabled=remove` is a super set operation. When applied, oci-monitor stops PX systemd service and then applies the `px/enabled=false` label on the node.

Reply:
Is it same for `px/enabled=uninstall` as well?

Reply:
We don't have the uninstall switch. `px/enabled=remove` is the switch that does the uninstall.